rustfmt.toml

```diff
@@ -1,18 +1,18 @@
 edition = "2021"
-fn_single_line = false
-format_code_in_doc_comments = true
-format_strings = true
-imports_layout = "HorizontalVertical"
-imports_granularity = "One"
-normalize_comments = true
-normalize_doc_attributes = true
+#fn_single_line = false
```

Comment:
Why removing these?

Reply:
They need `nightly`, I'd like to actually keep `stable` everywhere